control-plane/grpc/src/operations/nexus/traits.rs

```diff
@@ -370,6 +370,7 @@ impl TryFrom<nexus::NexusSpec> for NexusSpec {
                 .into_iter()
                 .map(TryInto::try_into)
                 .collect::<Result<_, _>>()?,
+            //rpolicy: RuleSet::assign(value.size),
```

Comment:
Let's just remove the commented code and add the actual code whenever we need to.